cocos/base/CCProperties.cpp

```diff
@@ -480,10 +480,8 @@ char* Properties::trimWhiteSpace(char *str)
         return str;
     }
 
-    char *end;
-
     // Trim leading space.
-    while (isspace(*str))
+    while (*str != '\0' && isspace(*str))
```

Comment:
But `isspace('\0')` will definitely return `false`, therefore it seems a wrong report by Xcode?